components/TxForm.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button } from "@chakra-ui/button";
+import { FormControl, FormLabel } from "@chakra-ui/form-control";
+import { Input, InputGroup, InputRightElement } from "@chakra-ui/input";
+import { Box, Flex } from "@chakra-ui/layout";
+import { Select } from "@chakra-ui/select";
+
+export const TxForm = ({ tokenList }: { tokenList: { name: string }[] }) => {
```

Comment:
May make sense to move the tokenlist to this component and remove this prop to avoid future refactor

Reply:
done!